src/test/java/picard/sam/FilterSamReadsTest.java

```diff
@@ -99,7 +99,7 @@ public void testPairedIntervalFilter(final String intervalFilename, final int ex
         sortedSamIdx.deleteOnExit();
 
         final SAMFileWriter writer = new SAMFileWriterFactory()
-                .setCreateIndex(true).makeBAMWriter(builder.getHeader(), false, inputSam);
+                .setCreateIndex(true).makeSAMWriter(builder.getHeader(), false, inputSam);
```

Comment:
Looks like the right move to me -- the test case explicitly claims to "Build a sam file for testing" (rather than a BAM file)